cmd/kubeadm/app/util/version.go

```diff
@@ -72,3 +72,12 @@ func KubernetesReleaseVersion(version string) (string, error) {
 	}
 	return "", fmt.Errorf("version %q doesn't match patterns for neither semantic version nor labels (stable, latest, ...)", version)
 }
+
+// KubernetesVersionToImageTag is helper function that replaces all
+// non-allowed symbols in tag strings with underscores.
+// Image tag can only contain lowercase and uppercase letters, digits,
+// underscores, periods and dashes.
+func KubernetesVersionToImageTag(version string) string {
```

Comment:
That's the case for current usage in one place where we know input. but in case of being public function of utils package, we never know what will be input but output will be tag in acceptable form. I'll add comment about known usage and semver.